.github/workflows/linux.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         key: linux-${{ matrix.image }}-${{ matrix.compiler }}-${{ matrix.config }}
 
     - name: Update Compiler (GCC)
-      if: ${{ matrix.image == 'ubuntu-18.04' && matrix.compiler == 'gcc' }}
+      if: ${{ matrix.image == 'ubuntu-20.04' && matrix.compiler == 'gcc' }}
```

Comment:
I don't believe we need this entire part of the matrix anymore, since the reason for it was to work around the ubuntu 18.04 image being too old.

alternatively, instead of dropping 18.04, maybe we need to change the version of clang being installed from 10 to 11?

Reply:
Both clang-10 and clang-11 are already installed. update-alternatives doesn't seem to be doing anything. I can just remove this part.

Reply:
this specific comment is about gcc, but i said clang by mistake.

Clang 10 and clang 11 are both installed on ubuntu 18.04? that's surprising to me, since they weren't when i first set it up.

Honestly i have no real interest in seeing 18.04 supported still. that's 5 years old.

So lets first start with removing anything referring to 18.04, like you already did.

Then if more stuff can get removed, that's great too.